src/app/pages/apps/components/installed-apps/app-notes-card/app-notes-card.component.html

```diff
@@ -4,19 +4,14 @@ <h3 mat-card-title>{{ 'Notes' | translate }}</h3>
   </mat-card-header>
   <mat-card-content>
     <div class="notes">
-      <div class="notes-list">
-        <div class="notes-item">
-          {{ 'Thank you for installing {app}' | translate: { app: app.name } }}
-        </div>
-        <div class="notes-item">
-          {{ 'Your release is named {name}' | translate: { name: app.chart_metadata.name } }}
-        </div>
-        <div class="notes-item">
-          {{ 'To learn more about the release, try:' | translate }}
-        </div>
-        <div class="notes-item">
-          {{ '$ mychart cli command' | translate }}
-        </div>
+      <div #notes class="notes-list">
+        <markdown [data]="app.info.notes"></markdown>
+      </div>
+      <div *ngIf="isCollapsed && showMoreLess" class="gradient"></div>
+      <div *ngIf="showMoreLess" class="more-less">
+        <span (click)="changeCollapsed()">
+          {{ isCollapsed ? 'View More' : 'Collapse' | translate }}
```

Comment:
This won't get picked by message extractor. Needs to be `('View More' | translate) : ('Collapse' | translate)`